rotkehlchen/exchanges/exchange.py

```diff
@@ -473,49 +473,39 @@ def query_income_loss_expense(
 
         return ledger_actions
 
-    def query_history_with_callbacks(
+    def query_history_with_fail_callback(
             self,
             start_ts: Timestamp,
             end_ts: Timestamp,
-            success_callback: ExchangeHistorySuccessCallback,
             fail_callback: ExchangeHistoryFailCallback,
     ) -> None:
         """Queries the historical event endpoints for this exchange and performs actions.
-
-        In case of success passes the result to success_callback.
+        The results are saved in the DB.
         In case of failure passes the error to failure_callback
         """
         try:
-            trades_history = self.query_trade_history(
+            self.query_trade_history(
                 start_ts=start_ts,
                 end_ts=end_ts,
                 only_cache=False,
             )
-            margin_history = self.query_margin_history(
+            self.query_margin_history(
                 start_ts=start_ts,
                 end_ts=end_ts,
             )
-            asset_movements = self.query_deposits_withdrawals(
+            self.query_deposits_withdrawals(
                 start_ts=start_ts,
                 end_ts=end_ts,
                 only_cache=False,
             )
-            # Query (and save in the DB) any ledger actions. They will be included in history l8er
             self.query_income_loss_expense(
                 start_ts=start_ts,
                 end_ts=end_ts,
                 only_cache=False,
             )
-            exchange_specific_data = self.query_exchange_specific_history(  # pylint: disable=assignment-from-none  # noqa: E501
+            self.query_exchange_specific_history(  # pylint: disable=assignment-from-none
```

Comment:
you removed the assignment so I think that the pylint disable comment is no longer needed here